### PR TITLE
chore: publish to JSR

### LINKED ISSUE #169: Publish deno_emit on JSR
Given that the deno community is slowly phasing more toward [JSR](https://jsr.io) and JSR doesn't support http imports (such as is currently used with deno.land/x) this package should be released on there for people to have access to it without having to rely on the http import (ie if you want to bundle it with another package)

### PR DESCRIPTION
**Pull request by iuioiua**
Closes #169

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -79,3 +83,7 @@ jobs:
           token: ${{ secrets.DENOBOT_PAT }}
           git-user-name: denobot
           git-user-email: denobot@users.noreply.github.com
+      
+      - name: Publish to JSR
+        if: contains(matrix.os, 'ubuntu')
+        run: cd js && deno run -A jsr:@david/publish-on-tag@0.1.3
```
Comment (dsherret) [inline .github/workflows/ci.yml:89]:
```suggestion
        run: cd js && deno run -A jsr:@david/publish-on-tag@0.1.3

```

File: deno.jsonc [hunk 2]
```diff
@@ -11,5 +11,6 @@
     "testdata",
     "target",
     "tests/__snapshots__"
-  ]
+  ],
+  "imports": { "@std/path": "jsr:@std/path@^0.223.0" }
```
Comment (dsherret) [inline deno.jsonc:15]:
```suggestion
  "imports": {
    "@std/path": "jsr:@std/path@^0.223.0"
  }
```

File: js/deno.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@deno/emit",
+  "version": "0.0.0",
+  "exports": "./mod.ts",
+  "publish": {
+    "exclude": [
+      "!**"
+    ]
+  },
+  "imports": { "@std/path": "jsr:@std/path@^0.223.0" }
```
Comment (dsherret) [inline js/deno.json:10]:
```suggestion
  "imports": {
    "@std/path": "jsr:@std/path@^0.223.0"
  }
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -79,3 +83,7 @@ jobs:
           token: ${{ secrets.DENOBOT_PAT }}
           git-user-name: denobot
           git-user-email: denobot@users.noreply.github.com
```
Comment (dsherret) [inline .github/workflows/ci.yml:85]: Let's remove this publish to /x and get tag version.

Comment (iuioiua) [reply .github/workflows/ci.yml]: Done

**Review by dsherret — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**